packages/manager/modules/pci/src/projects/project/storages/containers/users/delete/delete.controller.js

```diff
@@ -0,0 +1,44 @@
+import get from 'lodash/get';
+
+export default class PciStoragesContainersUsersDeleteController {
+  /* @ngInject */
+  constructor($http, $translate, PciProjectStorageContainersService) {
+    this.$http = $http;
+    this.$translate = $translate;
+    this.PciProjectStorageContainersService = PciProjectStorageContainersService;
+  }
+
+  $onInit() {
+    this.isDeleting = false;
+  }
+
+  delete() {
+    this.isDeleting = true;
+    return this.PciProjectStorageContainersService.removeAllCredentials(
+      this.projectId,
+      this.userId,
+      this.credentials,
+    )
+      .then(() =>
+        this.goToUsers(
+          this.$translate.instant(
+            'pci_projects_project_storages_containers_users_delete_success',
+            {
+              user: this.user.description,
+            },
+          ),
+        ),
+      )
+      .catch((error) =>
+        this.goToUsers(
```

Comment:
How do you handle partial success? few users deleted, few not. You need to inform user that some/all users deletion failed.